### ISSUE TITLE
akita can't train

### ISSUE THREAD

**Issue by bioczsun** [calico/basenji#160]
I retrained akita using the code you provided, but it does not work, out of memory. I rewrote some code, but pearsonr and loss oscillated back and forth. thank you!

**Comment by davek44** [comment]
Yes, Akita uses a lot of memory. I used GPUs with 24 Gb of RAM to train the published models.

It's hard for me to comment on how the rewriting you did might have affected the model and its training performance.
